### PR TITLE
feat: English on Mobile

### PR DESCRIPTION
**Pull request by Sembauke**
- **feat: english**
- **feat: a bunch stuff**

Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by Nirajn2311** [comment]
image

When the challenge view loads up the MCQ choices are not aligned properly. They align properly only after you scroll the page

**Review by Nirajn2311 — COMMENTED** [review]
image

When the challenge view loads up the MCQ choices are not aligned properly. They align properly only after you scroll the page

File: mobile-app/lib/models/learn/challenge_model.dart [hunk 1]
```diff
@@ -220,3 +233,59 @@ class ChallengeFile {
...
+  final String startTime;
+  final String startTimeStamp;
+  final String finishTimeStamp;
+
+  factory EnglishAudio.fromJson(Map<String, dynamic> data) {
+    return EnglishAudio(
+      fileName: data['filename'],
+      startTime: data['startTime'].toString(),
+      startTimeStamp: data['startTimestamp'].toString(),
+      finishTimeStamp: data['finishTimestamp'].toString(),
```
[the remark this answers is not in the document]
Comment (Sembauke) [reply mobile-app/lib/models/learn/challenge_model.dart:288]: Yes, it is easier to work with.

**Comment by Nirajn2311** [comment]
> > Details
> > When the challenge view loads up the MCQ choices are not aligned properly. They align properly only after you scroll the page
> 
> I don't know what to do about this. It seems to be an HTML issue.

Ok found the cause of the bug. When the view loads up initially, the audio is still loading and the spinner takes up more height than the progress bar so when the progress bar appears the whole UI gets a sudden shift up. But for some reason the text don't move up. For now I've put in a temporary fix

**Review by Nirajn2311 — APPROVED** [review]

**Pull request merged**